### PR TITLE
[DEV APPROVED] 8889 - Styleguide Component | teaser box

### PR DESCRIPTION
**Pull request by johnplayer1982**
# Styleguide Component | teaser box

[TP Ticket 8889](https://moneyadviceservice.tpondemand.com/entity/8918-styleguide-ui-component-teaser-box)

Add the teaser box UI component for the new fincap website.

image

### CODE REVIEW

File: app/views/styleguide/teaser.html.erb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <div class="teaser-box">
+      <img src="<image-url>" class="teaser-box__image" />
+      <h3 class="teaser-box__title">Title</h3>
+      <p class="teaser-box__content">
+        Content
+      </p>
+      <a href="#" class="teaser-box__cta">Call to action</a>
+    </div>
+  </div>
+</xmp>
```
Comment (Tr4pSt3R) [inline app/views/styleguide/teaser.html.erb]: @johnplayer1982 this closing tag needs attention. I think it's supposed to be `</div>`?

Comment (johnplayer1982) [reply app/views/styleguide/teaser.html.erb]: It's an example element, prevents the HTML from being rendered

Comment (Tr4pSt3R) [reply app/views/styleguide/teaser.html.erb]: Hmm, I've learnt something need Today! Thanks @johnplayer1982. All looks good but I'll wait on @davidtrussler 's lead :)

**Comment by davidtrussler** [comment]
👍  Looks good to me. 

Couple of comments (but can be considered later):
- We could use flexbox to ensure a constant height of all boxes where there are differing amounts of text.
- On the narrowest viewport before the boxes stack the individual elements are very narrow and may look odd. With a group of three elements though it's not easy to deal with this.

**Comment by johnplayer1982** [comment]
@davidtrussler Updated to use flexbox, that was a really good point!  The narrow boxes are a bit of an issue, but I feel its the lesser of 2 evils, the alternative means huge images at the largest size before the next breakpoint, Might be worth re-visiting once we have some pages with real content.

**Pull request merged**